src/backend/dx11/src/lib.rs

```diff
@@ -20,7 +20,6 @@ of continuous descriptor ranges (per type, per shader stage).
 extern crate bitflags;
 #[macro_use]
 extern crate log;
-#[macro_use]
 extern crate winapi;
```

Comment:
no need for `extern crate` declarations in Rust2018 if you aren't importing macros